src/ExtendedGridOrbitals.cc

```diff
@@ -250,7 +245,14 @@ void ExtendedGridOrbitals::initGauss(
     for (int icolor = 0; icolor < numst_; icolor++)
     {
         ORBDTYPE* ipsi = psi(icolor);
-        memset(ipsi, 0, numpt_ * sizeof(ORBDTYPE));
+        unsigned int const ipsi_size
+            = block_vector_.get_allocated_size_storage();
```

Comment:
I would use ipsi_size = numpt_ if you want to set one function/column at a time

Reply:
I've changed the size but it didn't matter. I don't understand what's the problem if the size is too large? I may copy too many items from/to the device but I don't see how this will give me a wrong result.

Reply:
I fixed the size and it allows the runs to pass that.
Then it fails in  ExtendedGridOrbitals::computeMatB() when some GPU <-> CPU code seems to be missing. I am running the test SiH4: mgmol -c tests/SiH4/mgmol.cfg -i tests/SiH4/sih4.xyz